pkg/common/constants/constants.go

```diff
@@ -133,10 +128,12 @@ const (
 	PublicCertsDir        = "/public-certs"
 
 	// DevWorkspace
-	DevWorkspaceControllerName     = "devworkspace-controller"
-	DevWorkspaceOperatorName       = "devworkspace-operator"
-	DevWorkspaceServiceAccountName = "devworkspace-controller-serviceaccount"
-	DefaultContainerBuildSccName   = "container-build"
+	DevWorkspaceControllerName             = "devworkspace-controller"
+	DevWorkspaceOperatorName               = "devworkspace-operator"
+	DevWorkspaceServiceAccountName         = "devworkspace-controller-serviceaccount"
+	DefaultContainerBuildSccName           = "container-build"
+	DefaultContainerRunSccName             = "container-run"
```

Comment:
@tolusha, wondering if we need to create a new SCC? maybe we can simply fall back on `restricted-v3` or `nested-container`, that are specifically designed for use with user namespaces according to the RNs - https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html-single/release_notes/index#ocp-release-notes-machine-config-operator-namespace_release-notes

Reply:
Sounds good, I have to test

Reply:
thanks, please check, but my understanding is that for the default arbitrary / random UID is used, and we want it to be 1000 for pre-configuring `/etc/subuid, /etc/subgid`

Reply:
Right, it does't work for our case.

<img width="1838" height="362" alt="image" src="https://github.com/user-attachments/assets/35a4db43-ea4c-458c-9941-869b6636ee05" />

Reply:
Yeah, I prefer to use a custom SCC that pre-determines the UID/GID of the workspace.  That allows you to lock down all of the /etc files instead of having to modify them at run time.

I updated my reference if that is helpful - https://github.com/cgruver/ocp-4-20-nested-containers